kyo-core/shared/src/test/scala/kyoTest/envTest.scala

```diff
@@ -0,0 +1,109 @@
+package kyoTest
+
+import kyo.*
+
+class envTest extends KyoTest:
```

Comment:
Can you add tests showing that getting a type not present in the `Env` fails to compile?

Reply:
Done.